bchrpc/bchrpc.proto

```diff
@@ -102,6 +130,7 @@ service bchrpc {
     // grpc-web.
     //
     // **Requires TxIndex to receive input metadata**
+    // TODO: Add slp metadata similar to "SubscribeTransactions"
```

Comment:
This still accurate? Should we expect it as part of this PR?

Reply:
Did this happen?

Reply:
I have removed the TODO note.  If the client wants token genesis metadata they can call GetTokenMetadata endpoint.